core/src/main/scala/pureconfig/error/ConfigReaderException.scala

```diff
@@ -12,42 +12,30 @@ final case class ConfigReaderException[T](failures: ConfigReaderFailures)(implic
     val linesBuffer = mutable.Buffer.empty[String]
     linesBuffer += s"Cannot convert configuration to a value of class ${ct.runtimeClass.getName}. Failures are:"
```

Comment:
Suggestion: "value of class" is a bit redundant here. It would be nicer if it read "Cannot convert configuration to a ${ct.runtimeClass.getName}", IMO.